ts/features/idpay/initiative/details/components/InitiativeCardComponent.tsx

```diff
@@ -23,133 +24,157 @@ type Props = {
+    flexDirection: "row",
+    alignItems: "center"
   },
   consumedOpacity: {
     opacity: 0.5
+  },
+  remainingPercentageSliderContainer: {
+    height: 4,
+    backgroundColor: IOColors.white,
+    width: 100,
+    borderRadius: 4
+  },
+  alignCenter: {
+    alignItems: "center"
   }
 });
 
+type PercentageSliderProps = {
+  percentage: number;
+  isGreyedOut: boolean;
+};
+const BonusPercentageSlider = ({
+  percentage,
+  isGreyedOut
+}: PercentageSliderProps) => {
+  const width = useSharedValue(0);
+  React.useEffect(() => {
+    // eslint-disable-next-line functional/immutable-data
+    width.value = percentage;
+  });
+  const scalingWidth = useAnimatedStyle(() => ({
+    width: withTiming(width.value, { duration: 1000 })
+  }));
+  return (
+    <View style={styles.remainingPercentageSliderContainer}>
+      <Animated.View
+        style={[
+          {
+            width: `${width.value}%`,
+            backgroundColor: isGreyedOut ? IOColors.blue : IOColors["grey-450"],
+            flex: 1,
+            borderRadius: 4
+          },
+          scalingWidth
+        ]}
+      />
+    </View>
+  );
+};
 const InitiativeCardComponent = (props: Props) => {
   const { initiativeName, endDate, status, amount, accrued, refunded } =
     props.initiative;
 
   const isInitiativeConfigured = status === InitiativeStatusEnum.REFUNDABLE;
   const toBeRepaidAmount = (accrued || 0) - (refunded || 0);
+  const remainingAmount = (amount || 0) - (accrued || 0);
 
-  const renderFullCard = () => (
-    <View style={[styles.row, styles.spaced, IOStyles.flex]}>
-      <View style={[styles.fullHeight, styles.spaced]}>
-        <View>
-          <VSpacer size={8} />
-          <H2 color="white">{initiativeName}</H2>
-          <Text style={{ color: IOColors.white }}>
-            {I18n.t("idpay.initiative.details.initiativeCard.validUntil", {
-              expiryDate: formatDateAsLocal(endDate, true)
-            })}
-          </Text>
-        </View>
-        <View style={styles.row}>
-          <View>
-            <Text
-              bold={true}
-              style={[
-                !isInitiativeConfigured ? styles.consumedOpacity : {},
-                styles.amount
-              ]}
-            >
-              {formatNumberAmount(amount || 0, true)}
-            </Text>
-            <H5 color="white">
+  const dateString = formatDateAsLocal(endDate, true);
+  const remainingBonusAmountPercentage =
+    amount !== 0 && amount !== undefined
+      ? (remainingAmount / amount) * 100.0
+      : 100.0;
+  return (
+    <View style={styles.cardContainer} testID={"card-component"}>
+      {/* top part */}
+      <View style={styles.topCardSection}>
+        <View style={styles.bonusLogoContainer}></View>
+        <VSpacer size={8} />
+        <H1>{initiativeName}</H1>
+        <Body>{/* the ministry would go here */}</Body>
+        <VSpacer size={8} />
+        <View style={styles.bonusStatusContainer}>
+          <Badge style={styles.badge}>
```

Comment:
Do not use the native-base `Badge`. Instead you could easily create a badge styled with just a background color and a border radius. 
Also, the badge has differences with the original design (bigger size and bigger font)

Original design:
<img width="549" alt="Screenshot 2023-03-18 alle 10 27 08" src="https://user-images.githubusercontent.com/6160324/226097307-f9eb9832-58fa-4515-af14-c36e65f2bc62.png">

Your design:
<img width="241" alt="Screenshot 2023-03-18 alle 10 27 50" src="https://user-images.githubusercontent.com/6160324/226097314-4122f5ad-fee4-4d5f-891d-dad1f0cd2eca.png">